src/jit/IntMath32Inl.h

```diff
@@ -1159,3 +1159,35 @@ static void emitMove64(sljit_compiler* compiler, Instruction* instr)
     sljit_emit_op1(compiler, SLJIT_MOV, dst.arg1, dst.arg1w, src.arg1, src.arg1w);
     sljit_emit_op1(compiler, SLJIT_MOV, dst.arg2, dst.arg2w, src.arg2, src.arg2w);
 }
+
+static void emitGlobalGet64(sljit_compiler* compiler, Instruction* instr)
+{
+    CompileContext* context = CompileContext::get(compiler);
+    Operand* operands = instr->operands();
+    JITArgPair dst;
+
+    operandToArgPair(operands, dst);
+
+    GlobalGet64* globalGet = reinterpret_cast<GlobalGet64*>(instr->byteCode());
+
+    sljit_emit_op1(compiler, SLJIT_MOV_P, SLJIT_R0, 0, SLJIT_MEM1(kContextReg), OffsetOfContextField(instance));
+    sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R0, 0, SLJIT_MEM1(SLJIT_R0), context->globalStart + globalGet->index() * sizeof(void*));
+    sljit_emit_op1(compiler, SLJIT_MOV, dst.arg1, dst.arg1w, SLJIT_MEM1(SLJIT_R0), JITFieldAccessor::globalValueOffset());
```

Comment:
Please use WORD_LOW_OFFSET / WORD_HIGH_OFFSET. This code is not big endian compatible.